app_dart/test/src/utilities/mocks.mocks.dart

```diff
@@ -155,87 +156,89 @@ class _FakeClientContext_49 extends _i1.Fake implements _i19.ClientContext {}
 
 class _FakePullRequestMerge_50 extends _i1.Fake implements _i9.PullRequestMerge {}
 
-class _FakePullRequestReview_51 extends _i1.Fake implements _i9.PullRequestReview {}
+class _FakePullRequestComment_51 extends _i1.Fake implements _i9.PullRequestComment {}
```

Comment:
I'm not sure why I needed to regenerate these to get the tests to pass, but it might be related to us updating dependabot to run on cocoon

Reply:
Yeah, I noticed that dependabot may be using a different version from what Dockerfile is using.